src/_main_/utils/route_handler/__init__.py

```diff
@@ -13,7 +13,7 @@ def __init__(self):
 
   def add(self, route: str, view: function) -> bool:
     path = route[1:]
-    self.routes[path] = view
+    self.routes[path] = timed(view) # for now we want to time every handler
```

Comment:
Do you expect this to add some latency, because of the exception handling within the timed method?

Reply:
No, for the following reasons:
1. We don't time every time.  For now we are doing 50% of the time.  It is a probabilistic thing.
2. The part of the code that does the computation and sending the cloudwatch is done in a spin-off thread